### PR TITLE
Add blog card

### PR DESCRIPTION
**Pull request by hirundinidae**
- missing white line between the blog title and tags, not sure how to add that 
- blog props set in blogSection, should change to use an array of blog posts (fetch from db) and map to blog cards 

screenshot_20210818_135523
screenshot_20210818_135516

### CODE REVIEW

File: components/BlogCard.jsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  <Wrapper {...props}>
+    <Row position="top" topbgcolor={topBgColor}>
+      <LeftSection>
+        <BlogTitle>{title}</BlogTitle>
+      </LeftSection>
+      <RightSection>
+        <TagSection tags={tags}>
+          {!!tags &&
+            tags.map((tag) => (
+              <Tag key={tag}>
```
Comment (xsharonhe) [inline components/BlogCard.jsx]: Abstract ```Tag``` into its own component

File: components/BlogCard.jsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    `
+      width: 90%; 
+    `
+  )};
+`;
+
+const Row = styled.div`
+  display: flex;
+  padding: 2rem;
+  border-radius: 100px;
```
Comment (xsharonhe) [inline components/BlogCard.jsx]: Hmm this property is overwritten in the next line

File: components/BlogCard.jsx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+  ${({ theme }) => `
+    background-color: ${theme.colors.greyBlue}; 
+  `};
+`;
+
+const SText = styled(Text)`
+  margin: 0;
+`;
+
+const AuthorSection = styled.div`
```
Comment (xsharonhe) [inline components/BlogCard.jsx:160]: I think ```AuthorSection``` and ```DayInfo``` probably should have the same alignment - it looks kinda weird with flex-end and then justify-content

File: pages/resources/blog.jsx [hunk 4]
```diff
@@ -72,6 +73,9 @@ export default function Blog({ blogs, ...props }) {
             value={searchParameter}
             onChange={handleChange}
           />
+          <BlogWrapper>
+            <BlogSection />
+          </BlogWrapper>
```
Comment (xsharonhe) [inline pages/resources/blog.jsx]: Feel free to actually replace this with the actual blog data down below with ```blogPosts```

File: components/BlogCard.jsx [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+        </AuthorSection>
+        <DayInfo>
+          <SText>{publishedDate}</SText>
+        </DayInfo>
+      </RightSection>
+    </Row>
+  </Wrapper>
+);
+
+const Wrapper = styled.div`
```
Comment (xsharonhe) [inline components/BlogCard.jsx:61]: Add a ```:hover``` property so that when you hover it looks different

**Review by xsharonhe — COMMENTED** [review]
Some minor comments

File: components/BlogCard.jsx (hunk 3 above)
Comment (hirundinidae) [reply components/BlogCard.jsx:160]: Changed to always left alignment

**Review by xsharonhe — COMMENTED** [review]
Should do for now. I think there can be small changes like automatically formatting the dates etc. and this really shows me we need to update some of those blog summaries

**Review by xsharonhe — APPROVED** [review]
I meant to approve whoops

**Comment by hirundinidae** [comment]
Yep, added a ticket to trello to format the dates.

**Pull request merged**